orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/SaltOrchestrator.java

```diff
@@ -819,6 +819,17 @@ public void removeDeadSaltMinions(GatewayConfig gateway) throws CloudbreakOrches
         }
     }
 
+    @Override
+    public Map<String, String> replacePatternInFileOnAllHosts(GatewayConfig gatewayConfig, String file, String pattern, String replace)
+            throws CloudbreakOrchestratorFailedException {
+        try (SaltConnector saltConnector = saltService.createSaltConnector(gatewayConfig)) {
+            return SaltStates.replacePatternInFile(retry, saltConnector, file, pattern, replace);
+        } catch (RuntimeException e) {
+            LOGGER.info("Error occurred during file replace execution in file '{}' while replacing pattern '{}' with '{}'", file, pattern, replace, e);
```

Comment:
I think it should be warn at least

Reply:
All other methods log the error with info, should I replace them all?

Reply:
I'm fine with this only, warn suits this better then info in my opinion